### DISCUSSION TITLE
Add "Count rows" to table right-click menu

### DISCUSSION THREAD

**Discussion by yonisade** [dbeaver/dbeaver#12370] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
It will be nice to have a built-in option to "Count rows" of a table like in TOAD/pgAdmin.

**Describe the solution you'd like**
Add this option to the right-click menu of a table and display row count on screen.

**Describe alternatives you've considered**
manually writing and executing: select count(*) from [table]

**Comment by p91paul** [comment, marked as answer]
at the bottom of the table you'll see a small refresh-like button which does what you want!
